### ISSUE TITLE
Add a description to the boot option in systemd-boot

### ISSUE THREAD

**Issue by Torxed** [archlinux/archinstall#1547]
Since [systemd 252 *(2022-10-31)*](https://github.com/systemd/systemd/blob/main/NEWS) the option `--efi-boot-option-description=` was added to `bootctl` which can:
> --efi-boot-option-description= to control the name of the boot entry.

We could utilize this to give a better name for the boot entries, to avoid multiple boot entries having the same name.
Currently it will default to something like "Arch Linux Boot loader" or "Linux Boot loader", and after a few installs it will be unclear which one is which. We could change it to "Arch Linux 2022-10-31" or something similar.

**Comment by ShaunTheSalvo** [comment]
Good idea. Is there some way you could allow the user to enter an open text string to be used for the boot entry, so they can name each install to something that's meaningful to them ("Arch test with XFCE 2022-11-11" or whatever). Or at least add a date stringto the boot entry.
